llvm/test/LTO/X86/codemodel-2.ll

```diff
@@ -1,5 +1,5 @@
 ; RUN: llvm-as %s -o %t.o
-; RUN: llvm-lto2 run -r %t.o,_start,px %t.o -o %t.s
+; RUN: llvm-lto2 run -r %t.o,_start,px -r %t.o,_GLOBAL_OFFSET_TABLE_, %t.o -o %t.s
```

Comment:
Why is it needed here and in other tests now?

Reply:
For x86-64 medium/large code model, the IR symbol table now references `_GLOBAL_OFFSET_TABLE_`, which requires llvm-lto2 to provide it to avoid a ` missing symbol resolution for ` error